acquisitions-unit/schemas/acquisitions_unit.json

```diff
@@ -40,6 +44,7 @@
   },
   "additionalProperties": false,
   "required": [
-    "name"
+    "name",
+    "isDeleted"
```

Comment:
It seems to me contradictory to make the field required and define the default value for it. I think we should remove something of this.

Reply:
@craigmcnally what is your opinion on this?

Reply:
As agreed, leaving this as already defined (required with default)